config/vufind/BrowZine.ini

```diff
@@ -23,10 +23,26 @@ default_view         = list
 ; setting. The default is "none," which performs no filtering.
 ;filterType = exclude
 ; This repeatable section can be used to filter based on link type; legal
-; options are "browzineWebLink" and "fullTextFile"
+; options are the keys defined in DOIServices section.
+; Note that this setting is available for convenience and compatibility with previous
+; versions. You can get the same results by changing the DOIServices section.
 ;filter[] = "browzineWebLink"
 ;filter[] = "fullTextFile"
 
+; This setting controls icons displayed with links. If set to false, icons are loaded
+; from Third Iron's assets. If true, icons defined in the DOIServices section are
+; used. Default is false.
+local_icons = false
+
+; This section defines the services to display from BrowZine and their display order.
+; Each key is a service name (see https://thirdiron.atlassian.net/wiki/x/WIDqAw for
+; more information) and value contains translation key, local icon and Third Iron's
+; icon asset (optional) separated by a pipe character.
+[DOIServices]
+browzineWebLink = "View Complete Issue|browzine-issue|https://assets.thirdiron.com/images/integrations/browzine-open-book-icon.svg"
+fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
+retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
```

Comment:
Do you think we should fall back on a local icon if now BrowZine icon URL is provided? Otherwise, by default, retraction notice has no icon at all.

Reply:
Makes sense, now added.